gammapy/spectrum/powerlaw.py

```diff
@@ -101,6 +102,37 @@ def power_law_flux(I=1, g=g_DEFAULT, e=1, e1=1, e2=E_INF):
     return I / _conversion_factor(g, e, e1, e2)
 
 
+def power_law_energy_flux(I, g=g_DEFAULT, e=1, e1=1, e2=10):
+    """
```

Comment:
For docstrings with math formulae, I suggest we always use "raw literal strings" (see [here](http://stackoverflow.com/a/2081708/498873)) to avoid having to write `\\` in the formula.
See example [here](https://gammapy.readthedocs.io/en/latest/_modules/gammapy/stats/poisson.html#significance)
I think we do this everywhere, if not I think we should.